pkg/api/frontendlogging/sentry.go

```diff
@@ -65,31 +64,28 @@ func (exception *FrontendSentryException) FmtStacktraces(store *SourceMapStore)
 	return strings.Join(stacktraces, "\n\n")
 }
 
-func addEventContextToLogContext(rootPrefix string, logCtx log15.Ctx, eventCtx map[string]interface{}) {
+func addEventContextToLogContext(rootPrefix string, logCtx *[]interface{}, eventCtx map[string]interface{}) {
```

Comment:
We should probably not use pointers to slices; slices are already pointers.

Reply:
that was my first try without pointer, but several lines later, i am pushing new element into slice, without pointer the unittest fails since the logCtx is not modified outside of the function. I am not sure under the hook how go is working in this case.

Reply:
If you need to modify the slice itself, then yes, pointer is needed.

Slices are reference types in that the contents of a slice are references to values, but not the slice itself.

Reply:
Would making it a custom type improve the readability? I.e. a struct with a slice in it, and pass a pointer to that struct? Or a type wrapper for a slice pointer with a custom Add() method?

Reply:
For methods if you go with that, https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/generic_vector.go has methods on slice pointers (e.g. Append) as syntax reference.

Reply:
I actually feel pointer to a vector is very readable instead of seeing some a custom name :D but it might just be me. If everyone agrees i would use the generic type instead.